automation/roles/prometheus/files/queries-default.yml

```diff
@@ -0,0 +1,120 @@
+---
+# {{ ansible_managed }}
+
+######################################################################
+# Copied from
+# https://github.com/wrouesnel/postgres_exporter/blob/1afbd62ab194c045a88488d77de2f116400dedb7/queries.yaml
```

Comment:
It looks like the data state of this commit was 8 years ago. Perhaps newer metrics should be used?

Latest version 0.17.1 / 2025-02-26

Reply:
let's update this part, we would not like to add old versions of metrics.